apps/example/tailwind.config.js

```diff
@@ -5,7 +5,7 @@
 
 module.exports = {
   presets: [
-    require("@cloudoperators/juno-ui-components/build/lib/tailwind.config"), // important, do not change
+    require("@cloudoperators/juno-ui-components/dist/lib/tailwind.config"), // important, do not change
```

Comment:
Renaming the build dir is a breaking change for all apps that depend on `juno-ui-components`. In this repo it's only the example and template app but we have all kinds of apps out in the world that will break if we make this change because they won't have access to the tailwind config that we provide to apps. This means they won't have access to our colors and other custom classes anymore.

Reply:
Gotcha, good catch! I thought about it too, I think it does not really matter build or dist, just on all examples in ts world I found dists, I thought if we will switch to typescript to make everything consistent. But I will make then build folder everywhere.